WISE-Community/WISE-Client
# chore(Teacher): Convert Milestone Grading View

## Changes

- Converted MilestoneGradingView to Angular
- Refactored NodeGradingView sort functions (MilestoneGradingView extends NodeGradingView)

## Test

1. Create a run (or use an existing run) that has a milestone with two locations (two CRater steps set up as before and after)
2. Have students submit work
3. Open the Classroom Monitor for the run and go to the Milestones view
4. Make sure the grading view for the first location works properly
  - Sort by team
  - Sort by status
  - Sort by score
  - Expand a workgroup
  - View revisions for a workgroup
  - Expand all
  - Collapse all
  - Search for student
  - Give a score
  - Give a comment
  - Make sure styling looks correct
5. Make sure the grading view for the second location works properly
  - Test same as first location but also
    - Sort by first location score
    - Sort by second location score
    - Sort by change in score
6. Make sure "Grade By Step" step view still sorts properly by Team, Status, and Score

Closes #820
